src/main/java/org/openstreetmap/atlas/checks/validation/tag/FixMeReviewCheck.java

```diff
@@ -0,0 +1,160 @@
+package org.openstreetmap.atlas.checks.validation.tag;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.openstreetmap.atlas.checks.base.BaseCheck;
+import org.openstreetmap.atlas.checks.flag.CheckFlag;
+import org.openstreetmap.atlas.geography.atlas.items.AtlasObject;
+import org.openstreetmap.atlas.geography.atlas.items.Edge;
+import org.openstreetmap.atlas.geography.atlas.walker.OsmWayWalker;
+import org.openstreetmap.atlas.tags.BuildingTag;
+import org.openstreetmap.atlas.tags.HighwayTag;
+import org.openstreetmap.atlas.tags.PlaceTag;
+import org.openstreetmap.atlas.tags.SurfaceTag;
+import org.openstreetmap.atlas.tags.WaterwayTag;
+import org.openstreetmap.atlas.tags.names.NameTag;
+import org.openstreetmap.atlas.tags.names.ReferenceTag;
+import org.openstreetmap.atlas.tags.oneway.OneWayTag;
+import org.openstreetmap.atlas.utilities.configuration.Configuration;
+
+/**
+ * This check flags features which contain the "fixme" or "FIXME" tag as well as other combinations
+ * of important tags.
+ *
+ * @author v-garei
+ */
+public class FixMeReviewCheck extends BaseCheck<Long>
+{
+    private static final long serialVersionUID = -2191776278923026805L;
+    private static final String HAS_FIXME_TAG = "Object {0, number, #} has 'fixme' tag and needs to be investigated.";
+    private static final List<String> FALLBACK_INSTRUCTIONS = Collections
+            .singletonList(HAS_FIXME_TAG);
+    private static final List<String> fixMeSupplementaryTags = List.of(WaterwayTag.KEY,
+            OneWayTag.KEY, BuildingTag.KEY, HighwayTag.KEY, NameTag.KEY, ReferenceTag.KEY,
+            PlaceTag.KEY, SurfaceTag.KEY);
+    private static final List<String> FIX_ME_SUPPORTED_VALUES_DEFAULT = List.of("verify",
+            "position", "resurvey", "Revisar: este punto fue creado por importación directa",
+            "continue", "name", "incomplete", "draw geometry and delete this point", "unfinished",
+            "recheck");
+    private final List<String> fixMeSupportedValues;
+    private final HighwayTag minHighwayTag;
+    private static final String MIN_HIGHWAY_TAG_DEFAULT = "tertiary";
+    private static final String fixMeLowerCase = "fixme";
```

Comment:
It would be great to have this as a Tag class in Atlas.

Reply:
I will add that. Wait to approve the PR until then?

Reply:
I'm happy to merge as is and then update the check when the tag is ready, or wait for the tag. If there isn't any rush on the check then it might be easier to keep track of if we wait.

Reply:
I would prefer to merge once this is ready because I will be moving to new tasks away from checks but can come back to this once the atlas repo is updated.